sdk/azidentity/logging.go

```diff
@@ -19,30 +18,6 @@ import (
 // used when obtaining credentials and the type of credential used.
 const EventAuthentication log.Event = "Authentication"
 
-// log environment variables that can be used for credential types
-func logEnvVars() {
```

Comment:
Why was this and the below logging removed?  Is it captured elsewhere?

Reply:
I removed this function because it was called only in the context of managed identity, and none of these variables apply to managed identity. `NewEnvironmentCredential` logs similar information. I removed `logMSIEnv` because `newManagedIdentityClient` now logs the same information.